### PR TITLE
Browser config

### LINKED ISSUE #61: add option to add default webpage to distro
Steps
- [x] Adding a url option to webapp
- [x] configuring meilix generator branch to use this url

### PR DESCRIPTION
**Pull request by xeon-zolt**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
to add a feature to able to add event url as default page of browsers

Fixes #61

### CODE REVIEW

**Comment by xeon-zolt** [comment]
test url --> https://meilix-generator-test2.herokuapp.com/

**Comment by abishekvashok** [comment]
> Event Link:
Can you make this more specific. Or add a `what is event url` with details on how it would affect the OS.

**Comment by xeon-zolt** [comment]
event url is the url of the event that will be set as default url like fossasia have a event and the url for the event is to be set as default home page for browsers

**Comment by abishekvashok** [comment]
@zeon-zolt i meant to make it clear to the end user :)

**Comment by xeon-zolt** [comment]
how can we do that ?
by adding help messuages in the form 
or can you suggest anything better

File: templates/index.html [hunk 1]
```diff
@@ -24,6 +24,12 @@ <h2 align="center">Meilix Generator</h2>
                         <label for="TRAVIS_TAG">Event Name:</label>
                         <input pattern="[A-Za-z0-9\s]{1,20}" class="form-control" placeholder="Enter event" name="TRAVIS_TAG" required="" type="text">
                     </div>
+                  
+                    <div class="form-group">
+                      <label for="event_url">Event Link:</label>
+                      <input pattern="https?://.+" class="form-control" placeholder="Enter event link" name="event_url" required="" type="url" >
```
Comment (meets2tarun) [inline templates/index.html]: Most of the websites are not SSL certified. So better we can go for http:// only.

**Comment by meets2tarun** [comment]
Please release an iso image and paste the link here, I will test that it takes the URL on home page or not.
Thanks @xeon-zolt

File: templates/index.html (hunk 1 above)
Comment (abishekvashok) [inline templates/index.html]: Can we get the user to decide.

**Comment by abishekvashok** [comment]
By adding on hover display messages.

File: templates/index.html (hunk 1 above)
Comment (xeon-zolt) [inline templates/index.html]: @meets2tarun  ok we can edit patter for that

**Comment by xeon-zolt** [comment]
@meets2tarun this part is only concerned with sending the url to travis its meilix duty to use the url variable

**Comment by mariobehling** [comment]
Please add a screenshot when you add a visual feature.

Follow best practices! Do not merge your own PRs!

**Comment by xeon-zolt** [comment]
screenshot from 2017-07-18 11-58-48
ok @mariobehling

**Pull request merged**